flow/project.py

```diff
@@ -1784,21 +1859,23 @@ def _fetch_scheduler_status(self, jobs=None, file=None, ignore_errors=False):
         "Update the status docs."
         if file is None:
             file = sys.stderr
-        if jobs is None:
-            jobs = list(self)
         try:
             scheduler = self._environment.get_scheduler()
 
             self.document.setdefault('_status', dict())
             scheduler_info = {sjob.name(): sjob.status() for sjob in self.scheduler_jobs(scheduler)}
             status = dict()
             print("Query scheduler...", file=file)
-            for job in tqdm(jobs,
-                            desc="Fetching operation status",
-                            total=len(jobs), file=file):
-                for group in self._groups.values():
-                    _id = group._generate_id((job,))
-                    status[_id] = int(scheduler_info.get(_id, JobStatus.unknown))
+            for group in tqdm(self._groups.values(),
+                              desc="Fetching operation status",
+                              total=len(self._groups), file=file):
+                aggregated_jobs = self.aggregates[group.name]
+                for aggregate in tqdm(aggregated_jobs, total=len(aggregated_jobs),
```

Comment:
Given that aggregates are _static_, we should revert back to job ordering. This is possible because we can just check that an aggregate is valid for a job. We just need to iterate over single jobs and then any user defined aggregation.

Reply:
Wouldn't this be too complicated? 
I don't have any problem with checking but I believe this will become a little bit complicated

Reply:
It would be slower granted. Before making changes, let's discuss this in the slack channel, and post our decision in the main PR GitHub discussion.

Reply:
The result of the discussion of slack on this topic is posted here
https://github.com/glotzerlab/signac-flow/pull/335#issuecomment-677747896